File: 242-ValidAnagram/242-ValidAnagram.cpp
// Last updated: 8/1/2025, 11:40:18 PM
class Solution {
public:
    bool isAnagram(string s, string t) {
        sort(s.begin(),s.end());
        sort(t.begin(),t.end());

        if(s==t){
            return true;
        }
        else{
            return false;
        }
        
    }
};

File: 242-ValidAnagram/242-ValidAnagram.py
# Last updated: 8/1/2025, 11:40:12 PM
class Solution:
    def isAnagram(self, s: str, t: str) -> bool:
        counter1 = {}
        counter2 = {}
        for char in s:
            if char in counter1:
                counter1[char]+=1
            else:
                counter1[char]=1
        for char in t:
            if char in counter2:
                counter2[char]+=1
            else:
                counter2[char]=1
        if counter1 == counter2:
            return True
        else:
            return False